### PR TITLE
Change session mode to resolve dirty output due to serial session

### PR DESCRIPTION
**Pull request by yanglei-rh**
1.Change session mode to resolve dirty output due to serial session

ID:2087909

Signed-off-by: Lei Yang <leiayang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
Test Pass
rhel.8.7:
(1/4) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.memory_leak_after_nichotplug.q35: PASS (131.68 s)
(2/4) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.msi_change_flag.vhostforce_on.disable_pci_msi.q35: PASS (162.23 s)
(3/4) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.bridge_vlan.q35: PASS (662.70 s)
(4/4) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.9.x86_64.io-github-autotest-qemu.nic_teaming.q35: PASS (271.95 s)

rhel9.1:
(1/4) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.bridge_vlan.q35: PASS (721.35 s)
(2/4) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.nic_teaming.q35: PASS (457.07 s)
(3/4) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.memory_leak_after_nichotplug.q35: PASS (164.92 s)
(4/4) Host_RHEL.m9.u1.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.x86_64.io-github-autotest-qemu.msi_change_flag.vhostforce_on.disable_pci_msi.q35: PASS (237.51 s)

**Comment by yanglei-rh** [comment]
Hello @PaulYuuu Could you please help me review it? Thanks in advance.

File: qemu/tests/cfg/nic_teaming.cfg [hunk 1]
```diff
@@ -1,6 +1,7 @@
 - nic_teaming: 
     virt_test_type = qemu
     only Linux
+    only bridge
```
Comment (PaulYuuu) [inline qemu/tests/cfg/nic_teaming.cfg:4]: OVS is also a bridge.

Comment (yanglei-rh) [reply qemu/tests/cfg/nic_teaming.cfg:4]: Sorry, I mistake it. So do you know any way to limit it to be tested only on the switch bridge? Thanks in advace.

Comment (yanglei-rh) [reply qemu/tests/cfg/nic_teaming.cfg]: I have resume this  file, it will be fixed other way in the future.

Comment (PaulYuuu) [reply qemu/tests/cfg/nic_teaming.cfg]: You can check if it's a Linux bridge or an OVS bridge.

**Pull request merged**